flavors/standard-Exasol-8-cuda-ml/flavor_base/build_run/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM {{build_deps}}
+
+RUN mkdir /exaudfclient /exaudf
+COPY exaudfclient/ /exaudfclient/
+
+WORKDIR /exaudfclient/
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$MAMBA_ROOT_PREFIX/lib/
+ENV CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH:$MAMBA_ROOT_PREFIX/include/
+ENV C_INCLUDE_PATH=$C_INCLUDE_PATH:$MAMBA_ROOT_PREFIX/include/
+RUN ./build.sh --config no-tty --config optimize --config python --config slow-wrapper
```

Comment:
`slow-wrapper` is only a temporary hack

Reply:
Will be fixed later in https://github.com/exasol/script-languages-release/issues/1032